eureka/src/main/java/com/linecorp/armeria/client/eureka/EurekaEndpointGroupBuilder.java

```diff
@@ -183,29 +184,46 @@ public EurekaEndpointGroupBuilder secureVipAddress(String secureVipAddress) {
     }
 
     /**
-     * Sets the interval between fetching registry requests. {@value #DEFAULT_REGISTRY_FETCH_INTERVAL_SECONDS}
-     * is used by default and it's not recommended to modify this value. See
+     * Sets the interval between fetching registry requests. {@value #DEFAULT_REGISTRY_FETCH_INTERVAL_MILLIS}
+     * milliseconds is used by default and it's not recommended to modify this value. See
      * <a href="https://github.com/Netflix/eureka/wiki/Understanding-eureka-client-server-communication#fetch-registry">
      * fetch-registry</a>.
      */
     public EurekaEndpointGroupBuilder registryFetchInterval(Duration registryFetchInterval) {
         requireNonNull(registryFetchInterval, "registryFetchInterval");
-        final long seconds = registryFetchInterval.getSeconds();
-        checkArgument(seconds > 0, "registryFetchInterval.getSeconds(): %s (expected: > 0)", seconds);
-        return registryFetchIntervalSeconds(seconds);
+        checkArgument(!registryFetchInterval.isZero() &&
+                      !registryFetchInterval.isNegative(),
+                      "registryFetchInterval: %s (expected: > 0)",
+                      registryFetchInterval);
+        return registryFetchIntervalMillis(registryFetchInterval.toMillis());
     }
 
     /**
-     * Sets the interval between fetching registry requests in seconds.
-     * {@value #DEFAULT_REGISTRY_FETCH_INTERVAL_SECONDS} is used by default and it's not recommended to modify
-     * this value. See
+     * Sets the interval between fetching registry requests in seconds. {@code 30} is used by default and
+     * it's not recommended to modify this value. See
      * <a href="https://github.com/Netflix/eureka/wiki/Understanding-eureka-client-server-communication#fetch-registry">
      * fetch-registry</a>.
+     *
+     * @deprecated Use {@link #registryFetchIntervalMillis(long)}.
      */
+    @Deprecated
     public EurekaEndpointGroupBuilder registryFetchIntervalSeconds(long registryFetchIntervalSeconds) {
         checkArgument(registryFetchIntervalSeconds > 0, "registryFetchIntervalSeconds: %s (expected: > 0)",
                       registryFetchIntervalSeconds);
-        this.registryFetchIntervalSeconds = registryFetchIntervalSeconds;
+        return registryFetchIntervalMillis(TimeUnit.SECONDS.toMillis(registryFetchIntervalSeconds));
+    }
+
+    /**
+     * Sets the interval between fetching registry requests in milliseconds.
+     * {@value #DEFAULT_REGISTRY_FETCH_INTERVAL_MILLIS} is used by default and it's not recommended to modify
```

Comment:
nit:
```suggestion
     * {@value #DEFAULT_REGISTRY_FETCH_INTERVAL_MILLIS} milliseconds is used by default and it's not recommended to modify
```

Reply:
I intentionally omitted it because I mentioned the value is in millis in the first sentence. Would it be better repeating it?